app/views/order_mailer/welcome_email.text.erb

```diff
@@ -0,0 +1,16 @@
+
+Welcome to example.com, <%= @user.name %>
+===============================================
+
+You have successfully signed up to example.com,
+your username is: <%= @user.login %>.
+
+To login to the site, just follow this link: <%= @url %>.
+
+Thanks for joining and have a great day!
+
+
+<!--What we need-->
+<!--customer email-->
+<!--time of the order-->
+<!--order details with dish name, ingredient and quantity-->
```

Comment:
Not sure if this one is needed

Reply:
this was just a reminder on what we needed and will be removed soon.